library/Zend/Http/Header/AbstractAccept.php

```diff
@@ -167,6 +167,11 @@ protected function getParametersFromFieldValuePart($fieldValuePart)
             }
 
             foreach ($paramsStrings as $param) {
+                // Prevents none well formated param
+                if (strstr($param, '=') === false) {
```

Comment:
http://php.net/strstr
> Note:
If you only want to determine if a particular needle occurs within haystack, use the faster and less memory intensive function strpos() instead.